usersimcrs/dialogue_management/dialogue_state.py

```diff
@@ -1,8 +1,8 @@
 """Representation of the dialogue state.
 
-The dialogue state includes the turn count, a list of dialogue acts per turn for
-both the agent and the user, and the belief state. The belief state is a
-dictionary that maps slot names to slot values.
+The dialogue state includes the number of utterance, a list of dialogue acts per utterance for both
```

Comment:
Nit: "number of utterances" and check line length

Reply:
Done.